en/flight_modes/land.md

```diff
@@ -0,0 +1,43 @@
+# LAND Flight Mode
+
+The LAND flight mode causes the vehicle to land at the position where the mode was engaged.
+
+The specific behaviour for each vehicle type is described below.
+
+> **Note** 
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+
+## Multi-Copter (MC)
+
+The vehicle will land at the location at which the mode was engaged. The vehicle descends at the rate specified in `MPC_LAND_SPEED` until it hits the ground.
+
+Landing is affected by the following parameters:
+
+Parameter | Description
+--- | ---
+[MPC_LAND_SPEED](../advanced_config/parameter_reference.md#MPC_LAND_SPEED) | The rate of descent during landing. This should be kept fairly low as the ground conditions are not known.
+
+## Fixed Wing (FW)
+
+The vehicle turns and lands at the location at which the mode was engaged. 
+
+> **Note** The vehicle doesn't necessarily know where the ground is, and will therefore follow a fixed trajectory for the whole landing approach (no flare).
+
+Landing is affected by the following parameters:

```

Comment:
This is where it gets convoluted. The autopilot likely doesn't know the actual height of the ground. When you jump into LAND as a mode it's the same landing behavior, but it internally created the land waypoint at altitude 0. As a result you probably aren't going to hit the flare logic (FW_LAND_FLALT) unless you actually get to FW_LAND_FLALT over the land altitude. If you have a distance sensor (and it's enabled), then it will be used instead.

If we used HOME altitude it would sometimes be right, sometimes be wrong.

It seems like we should be explaining FW LAND logic somewhere, and then reference it for LAND, MISSION, and RTL.